packages/svelte-vega/src/lib/utils.ts

```diff
@@ -1,5 +1,5 @@
 import type { View, SignalListeners } from './types';
-import { vega } from 'vega-embed';
+import { vega } from 'vega-embed/build/vega-embed.module.js';
```

Comment:
That doesn't feel right as it requires a specific file in a specific place.

The right fix seem to add import maps.